cmd/rook/ceph/osd.go

```diff
@@ -206,21 +206,21 @@ func writeOSDConfig(cmd *cobra.Command, args []string) error {
 
 	clientset, _, _, err := rook.GetClientset()
 	if err != nil {
-		rook.TerminateFatal(fmt.Errorf("failed to init k8s client. %+v\n", err))
+		rook.TerminateFatal(fmt.Errorf("failed to init k8s client %+v", err))
```

Comment:
The pattern rook follows for error messages is that the error is appended as a new sentence so we need the period.

Reply:
In kubernetes we follow a common pattern for error strings like , for example , above will transform to,
"failed to init k8s client, err: %+v", err"

I can change to this format if are in agreement in a follow up PR.